_starter-kit/_patterns/03-components/fieldset/_fieldset.scss

```diff
@@ -1,12 +1,17 @@
 // @file
 // Styles for form fieldsets.
 
+$base-line-height: 1.5;
```

Comment:
Should probably rename these two variables if they are now specific to this component.

Reply:
The field and fieldset files contain a lot of self-contained variables fonts and mixins. I would like to address them in separate ticket once we resolve the config files and approach